featureFlag/style.css

```diff
@@ -19,6 +19,10 @@ body {
   align-items: center;
 }
 
+button {
```

Comment:
Can we use `class` name instead of `tag`?

Reply:
Yes, we can but buttons are defined at the top level and could be overridden very easily. If new buttons or input with submit type is added, they will automatically have a pointer cursor and could be changed by defining using their class or id.

Reply:
If you are setting a universal properties for all buttons and elements then you can create a global CSS file also

Reply:
got it thanks 👍

Reply:
@sahsisunny I did the changes you asked